### PR TITLE
fix : (RDCC-901) validation for organisation

### PR DESCRIPTION
**Pull request by shreedhar-hmcts**
### JIRA link (if applicable) ###
RDCC-901

### Change description ###
validate organisation if resend invite user present in it

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: src/functionalTest/java/uk/gov/hmcts/reform/professionalapi/ReInviteUserTest.java [hunk 1]
```diff
@@ -151,4 +152,20 @@ public void should_reinvite_super_user_within_one_hour() {
             assertThat((String) reinviteUserResponse.get("errorDescription")).contains(String.format("The request was last made less than %s minutes. Please try after some time"), resendInterval);
         }
     }
+
+    // should not re invite when user does not exists in organisation
+    @Test
+    public void should_return_403_when_reinvited_user_not_exists_in_same_organisation() {
+        if (resendInviteEnabled) {
+
+            OrganisationCreationRequest organisationCreationRequest1 = someMinimalOrganisationRequest().build();
```
Comment (KOTLAPRA23) [inline src/functionalTest/java/uk/gov/hmcts/reform/professionalapi/ReInviteUserTest.java:161]: could you please make it more specific like internal and external scenario

File: src/integrationTest/java/uk/gov/hmcts/reform/professionalapi/ReInviteUserIntegrationTest.java [hunk 2]
```diff
@@ -130,4 +132,25 @@ public void should_return_409_when_reinvited_user_gets_active_in_sidam_but_pendi
...
     }
+
+    // should not allow re invite of user who is not in organisation
+    @Test
+    public void should_return_403_when_reinvited_user_not_present_in_organisation() {
+
+        if (resendInviteEnabled) {
+            userProfileCreateUserWireMock(HttpStatus.CREATED);
+
+            OrganisationCreationRequest organisationCreationRequest1 = someMinimalOrganisationRequest().build();
```
Comment (KOTLAPRA23) [inline src/integrationTest/java/uk/gov/hmcts/reform/professionalapi/ReInviteUserIntegrationTest.java:143]: could you  please add one more for external as-well and tend of the test case add separate it with internal and external scenarios

Comment (shreedhar-hmcts) [reply src/integrationTest/java/uk/gov/hmcts/reform/professionalapi/ReInviteUserIntegrationTest.java:143]: done

File: src/functionalTest/java/uk/gov/hmcts/reform/professionalapi/ReInviteUserTest.java (hunk 1 above)
Comment (shreedhar-hmcts) [reply src/functionalTest/java/uk/gov/hmcts/reform/professionalapi/ReInviteUserTest.java:166]: done

**Pull request merged**